bb/master/master.cfg

```diff
@@ -232,45 +274,62 @@ c["services"] = [
 c["change_source"] = []
 
 
-def pull_request_filter(pull_request_message):
-    # Include only changes from fork repositories
-    if pull_request_message["head"]["repo"] and pull_request_message["head"]["repo"][
-        "full_name"] == config.GITHUB_REPOSITORY:
+def is_msdk_org_member(pull_request_message):
+    commit_owner = pull_request_message['head']['user']['login']
+    github_member_url = f"https://api.github.com/orgs/Intel-Media-SDK/members/{commit_owner}?access_token={config.GITHUB_TOKEN}"
```

Comment:
Hardcode. Please, move to a common part or add TODO.

Reply:
Done